simulation_parameters/microbe_stage/organelles.json

```diff
@@ -1083,6 +1084,57 @@
     "EditorButtonGroup": "Organelle",
     "EditorButtonOrder": 5
   },
+  "ferroplast": {
+    "MPCost": 45,
+    "InitialComposition": {
+      "ammonia": 2,
+      "phosphates": 2
+    },
+    "Hexes": [
+      {
+        "r": 0,
+        "q": 0
+      },
+      {
+        "r": -1,
+        "q": 0
+      }
+    ],
+    "Processes": {
+      "ferrosynthesis": 1
+    },
+    "Components": {
+      "Storage": {
+        "Capacity": 1
+      }
+    },
+    "Enzymes": {
+      "rusticyanin": 1
+    },
+    "ProkaryoteChance": 0,
+    "ChanceToCreate": 1,
+    "Density": 1100,
+    "Graphics": {
+      "ScenePath": "res://assets/models/organelles/Ferroplast.tscn"
+    },
+    "Name": "FERROPLAST",
+    "ProductionColour": "#3293f7",
+    "ConsumptionColour": "#f04681",
+    "IconPath": "res://assets/textures/gui/bevel/parts/FerroplastIcon.png",
+    "UnlockConditions": [
+      {
+        "ReproduceWithOrganelle": {
+          "Organelle": "rusticyanin",
+          "Generations": 7,
+          "InARow": true
+        }
+      }
+    ],
+    "RequiresNucleus": true,
+    "EditorButtonGroup": "Organelle",
+    "EditorButtonOrder": 5,
+    "LAWK": false
```

Comment:
Could we make this a LAWK organelle? I'd like that because I'm nearly certain that endosymbiosis would be able to place this anyway, so if this is a non-LAWK organelle there's going to be future bug reports and I don't want to have to deal with those.

Reply:
Okay. I was going to suggest allowing it to be unlocked through endosymbiosis even if it couldn't be unlocked another way, though it would probably cause confusion if it could be accessed in some ways an not others.

Reply:
I can almost guarantee that someone would notice that, maybe years down the line and a bug report was opened.